lms/static/js/student_account/components/StudentAccountDeletionModal.jsx

```diff
@@ -31,7 +31,8 @@ class StudentAccountDeletionConfirmationModal extends React.Component {
     this.props.onClose();
 
     removeLoggedInCookies();
-    window.location.href = 'https://www.edx.org';
+    // Appsembler: Changed this from edx.org to our website
+    window.location.href = 'https://www.appsembler.com';
```

Comment:
Learners could be confused when redirected from a customer site to Appsembler's! 

How about:

```suggestion
window.location.href = location.protocol + '//' + location.host + '/';
```

For more information checkout the Mozilla Developer Network:

 - https://developer.mozilla.org/en-US/docs/Web/API/Location

Reply:
This definitely makes more sense. Fully agree with @OmarIthawi